Tasks/Common/PowershellHelpers/module.json

```diff
@@ -2,6 +2,7 @@
     "messages": {
         "ActionTimedOut": "Exceeded max retries. Timing out.",
         "RetryAfterMessage": "Retrying after {0} second(s)...",
-        "ActionException": "Exception occurred: {0}"
+        "ActionException": "Exception occurred: {0}",
```

Comment:
Where is this used?

Reply:
In Catch block of Invoke-ActionWithRetries